src/code42cli/cmds/search/output_processor.py

```diff
@@ -0,0 +1,34 @@
+import click
+
+from code42cli.logger import get_logger_for_server
+
+
+def print_events(format_function, output_header):
+    """Prints events to stdout"""
+
+    def decorator(events):
+        def paginate():
+            yield format_function(events, output_header)
+
+        if len(events) > 10:
+            click.echo_via_pager(paginate)
+        else:
+            for page in paginate():
+                click.echo(page)
+
+    return decorator
+
+
+def send_events(output_format, hostname, protocol, output_header, format_function):
+    """Sends events to server/hostname"""
+
+    def decorator(events):
+        def paginate():
+            yield format_function(events, output_header)
```

Comment:
This is the reason for your bug.

Do this `yield events` instead.

The formatting takes place at the logging level.  We are instead doing it twice here which causes the second time to error out.

Reply:
However, if you temporarily change this part to be `yield events` and line 32 to 
```
for page in paginate():
    for event in page:
        logger.info(event)
```
you would see it working